autonomi/tests/register.rs

```diff
@@ -22,7 +22,7 @@ use tokio::time::sleep;
 async fn register() -> Result<()> {
     let _log_appender_guard = LogBuilder::init_single_threaded_tokio_test("register", false);
 
-    let client = Client::init_local().await?;
+    let client = Client::init_local(true).await?;
```

Comment:
It's not 100% clear what the `true` here means, if an option on/off is needed maybe another constructor or the following pattern:
```
Client::init_local().with_option_AAA().await?;
```